src/domain/UBGraphicsTextItem.cpp

```diff
@@ -139,6 +139,44 @@ QFont UBGraphicsTextItem::createDefaultFont()
     return font;
 }
 
+bool UBGraphicsTextItem::event(QEvent *ev)
+{
+    if(ev->type() == QEvent::ShortcutOverride && isSelected())
+    {
+        QKeyEvent* keyEvent = static_cast<QKeyEvent*>(ev);
+        QKeySequence::StandardKey shortcuts[] = {QKeySequence::StandardKey::Bold,
+                                                 QKeySequence::StandardKey::Italic,
+                                                 QKeySequence::StandardKey::Underline};
+        QKeySequence::StandardKey key = QKeySequence::StandardKey::UnknownKey;
+        for(int i=0; i<3; i++)
+            if(keyEvent->matches(shortcuts[i]))
+            {
+                key = shortcuts[i];
+                break;
+            }
+        if(key != QKeySequence::StandardKey::UnknownKey)
+        {
+            QTextCursor curCursor = textCursor();
+            QTextCharFormat format;
+            QFont ft(curCursor.charFormat().font());
+            if(key == QKeySequence::StandardKey::Bold)
+                ft.setBold(!ft.bold());
+            else if(key == QKeySequence::StandardKey::Italic)
+                ft.setItalic(!ft.italic());
+            else
+                ft.setUnderline(!ft.underline());
+            format.setFont(ft);
+            curCursor.mergeCharFormat(format);
+            setTextCursor(curCursor);
+
+            contentsChanged();
+            ev->accept();
+            return true;
```

Comment:
Are you sure we want to avoid base class call ?

Reply:
My understanding of the `keyPressEvent()` documentation leads me to remove the `accept()` call and do not call the base class method:

> Note that [QKeyEvent](https://doc.qt.io/qt-6/qkeyevent.html) starts with isAccepted() == true, so you do not need to call [QKeyEvent::accept](https://doc.qt.io/qt-6/qevent.html#accept)() - just do not call the base class implementation if you act upon the key.

https://doc.qt.io/qt-6/qwidget.html#keyPressEvent

It seems we do not call the base class method if we intercept event also for the `event()` method. Have a look at the example of the documentation:
https://doc.qt.io/qt-6/qobject.html#event